lighthouse-core/test/config/config-test.js

```diff
@@ -556,7 +556,7 @@ describe('Config', () => {
     assert.deepStrictEqual(config.settings.output, ['html']);
   });
 
-  it('extends the full config', () => {
+  it('extends the config', () => {
```

Comment:
Should we leave in a test for compatibility with `lighthouse:full` (with a note that it can be removed in the future if we ever add general extending)? Support remains by default since any string will work, so maybe it's overkill, but at least a few folks are using it (like the pubads wrapper project).